.github/workflows/test.yml

```diff
@@ -49,13 +49,13 @@ jobs:
     - name: Read performance delta
```

Comment:
Why is the yml changing in this PR?

Reply:
It is gone now, as it was already also on `master`.